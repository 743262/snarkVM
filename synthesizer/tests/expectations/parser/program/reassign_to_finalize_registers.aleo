program reassign_to_function_registers.aleo;

function hello:
    input r0 as u32.public;
    input r1 as u32.public;
    finalize r0 r1;

finalize hello:
    input r0 as u32.public;
    input r1 as u32.public;
    add r0 r1 into r2;
    add r1 r2 into r2;
