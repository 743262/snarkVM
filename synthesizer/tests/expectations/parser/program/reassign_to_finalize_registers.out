Failed to parse string. Remaining invalid string is: "output r2 as u32.private;
"